dev/benchmarks/microbenchmarks/lib/foundation/platform_asset_bundle.dart

```diff
@@ -0,0 +1,46 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart' show PlatformAssetBundle;
+import 'package:flutter/widgets.dart';
+
+import '../common.dart';
+
+const int _kBatchSize = 100;
+const int _kNumIterations = 100;
+
+void main() async {
+  assert(false,
+      "Don't run benchmarks in debug mode! Use 'flutter run --release'.");
+  final BenchmarkResultPrinter printer = BenchmarkResultPrinter();
+  WidgetsFlutterBinding.ensureInitialized();
+  final Stopwatch watch = Stopwatch();
+  final PlatformAssetBundle bundle = PlatformAssetBundle();
+
+  final List<double> values = <double>[];
+  for (int j = 0; j < _kNumIterations; ++j) {
+    double tally = 0;
+    watch.reset();
+    watch.start();
+    for (int i = 0; i < _kBatchSize; i += 1) {
+      tally += (await bundle.load(
+              'packages/flutter_gallery_assets/places/india_pondicherry_salt_farm.png'))
```

Comment:
nit: consider adding a comment explaining that we don't load images anymore, but that images are easily available in this benchmark. Other realistic use cases for this would be things like Flare/Rive animations or a license file.

Also, please make this one line.

Reply:
done